### ISSUE TITLE
 I don't know why the fusion part is like this torch.cat((visfts, att_visual_features), 1)

### ISSUE THREAD

**Issue by wonjin0403** [praveena2j/JointCrossAttentional-AV-Fusion#4]
Hi 

I read the paper with interest!!

I think the fusion part should look like this, shouldn't it?

torch.cat((visfts, att_visual_features), 1) -> torch.cat((att_audio_features, att_visual_features), 1) 

Thank you.

**Comment by praveena2j** [comment]
Thanks for your interest in the paper.

Yes you are right,  i will update it

please find the updated link of code for the below.

https://github.com/praveena2j/Joint-Cross-Attention-for-Audio-Visual-Fusion
---------------------------------
Best Regards
Praveen



GOD
ALMIGHTY
True happiness is not found in any other reward than that of being united
with God..................
